package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@cloudflare/blindrsa-ts",
-    "version": "0.1.2",
+    "version": "0.1.3",
```

Comment:
don't update the version without a release, as you don't know which commits are part of it yet

Reply:
correct, fixed now